aws/resource_aws_ssm_activation.go

```diff
@@ -57,6 +57,7 @@ func resourceAwsSsmActivation() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"tags": tagsSchemaForceNew(),
```

Comment:
When adding new schema attributes we generally recommend that a new acceptance be added to validate the functionality especially for attributes with `ForceNew: true`. 

Something like the code below
```go
  func TestAccAWSSSMActivation_update(t *testing.T) {                                                                                                                                                                                                                                                 
    var ssmActivation1, ssmActivation2 ssm.Activation                                                                                                                                                                                                                                                 
    name := acctest.RandString(10)                                                                                                                                                                                                                                                                    
    resource.ParallelTest(t, resource.TestCase{                                                                                                                                                                                                                                                       
      PreCheck:     func() { testAccPreCheck(t) },                                                                                                                                                                                                                                                    
      Providers:    testAccProviders,                                                                                                                                                                                                                                                                 
      CheckDestroy: testAccCheckAWSSSMActivationDestroy,                                                                                                                                                                                                                                              
      Steps: []resource.TestStep{                                                                                                                                                                                                                                                                     
        {                                                                                                                                                                                                                                                                                             
          Config: testAccAWSSSMActivationBasicConfig(name, "My Activation"),                                                                                                                                                                                                                          
          Check: resource.ComposeTestCheckFunc(                                                                                                                                                                                                                                                       
            testAccCheckAWSSSMActivationExists("aws_ssm_activation.foo", &ssmActivation1),                                                                                                                                                                                                            
            resource.TestCheckResourceAttrSet("aws_ssm_activation.foo", "activation_code"),                                                                                                                                                                                                           
            resource.TestCheckResourceAttr("aws_ssm_activation.foo", "tags.%", "1"),                                                                                                                                                                                                                  
            resource.TestCheckResourceAttr("aws_ssm_activation.foo", "tags.Name", "My Activation"),                                                                                                                                                                                                   
          ),                                                                                                                                                                                                                                                                                          
        },                                                                                                                                                                                                                                                                                            
        {                                                                                                                                                                                                                                                                                             
          Config: testAccAWSSSMActivationBasicConfig(name, "Foo"),                                                                                                                                                                                                                                    
          Check: resource.ComposeTestCheckFunc(                                                                                                                                                                                                                                                       
            testAccCheckAWSSSMActivationExists("aws_ssm_activation.foo", &ssmActivation2),                                                                                                                                                                                                            
            resource.TestCheckResourceAttrSet("aws_ssm_activation.foo", "activation_code"),                                                                                                                                                                                                           
            resource.TestCheckResourceAttr("aws_ssm_activation.foo", "tags.%", "1"),                                                                                                                                                                                                                  
            resource.TestCheckResourceAttr("aws_ssm_activation.foo", "tags.Name", "Foo"),                                                                                                                                                                                                             
          ),                                                                                                                                                                                                                                                                                          
        },                                                                                                                                                                                                                                                                                            
      },                                                                                                                                                                                                                                                                                              
    })                                                                                                                                                                                                                                                                                                
  }                                                                                                                                                                                                                                                                                                   
                                                                                                                                                                                                                                                                                                      
  func testAccCheckAWSSSMActivationRecreated(t *testing.T, before, after *ssm.Activation) resource.TestCheckFunc {                                                                                                                                                                                    
    return func(s *terraform.State) error {                                                                                                                                                                                                                                                           
      if *before.ActivationId == *after.ActivationId {                                                                                                                                                                                                                                                
        t.Fatalf("expected SSM activation Ids to be different but got %v == %v", before.ActivationId, after.ActivationId)                                                                                                                                                                             
      }                                                                                                                                                                                                                                                                                               
      return nil                                                                                                                                                                                                                                                                                      
    }                                                                                                                                                                                                                                                                                                 
  } 
```